addons/account/models/account_move.py

```diff
@@ -1164,7 +1163,7 @@ def _compute_payment_state(self):
                                 new_pmt_state = invoice._get_invoice_in_payment_state()
 
                         else:
-                            new_pmt_state = 'paid'
+                            new_pmt_state = 'paid' if invoice.state == 'posted' else 'not_paid'
```

Comment:
So even if the invoice is fully paid in draft, you display it as to be paid? This looks like a buggy behavior

Reply:
No
This is simply because we do not want to consider a draft invoice with 0 `amount_residual` (and no `has_st_line` nor `has_payment`)